jaxrs-tck/pom.xml

```diff
@@ -90,12 +90,30 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.3.2</version>
             </plugin>
+            <!--plugin>
```

Comment:
This recording of the signature file would better be done in a profile

Reply:
Moved the signature file generation to profile `record-signature`

Reply:
> Moved the signature file generation to profile `record-signature`. Can use `mvn clean install -Precord-signature` to generate signature file along with tck build.

Ping @scottkurz ^, looks like Jakarta RESTful Web Services is also generating TCK signatures via maven with https://github.com/jtulach/netbeans-apitest, like I think Jakarta Batch is doing.  Making the connection here in  case there is anything (knowledge, issues) to share.

Reply:
Hi.  I think I proved the concept but don't yet have it packaged to refer to.   I opened a few PRs: https://github.com/jtulach/netbeans-apitest/pulls  on the tool.  No idea when to expect they might be merged.  But I can work around  them for EE 10.    Scott M. and I shared some observations here:  https://www.eclipse.org/lists/jakartaee-tck-dev/msg01439.html